frontend/src/app/Notification.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useNotifications } from "./NotificationContext";
+import "./Notifications.css";
+
+function Notification() {
+  const { notifications, setNotifications } = useNotifications();
+  const [parsedNotifs, setParsedNotifs] = useState([]);
+
+  useEffect(() => {
+    setParsedNotifs([]);
+    for (const notif in notifications) {
+      setParsedNotifs((prev) => [
+        ...prev,
+        JSON.parse(notifications[notif].data),
+      ]);
+    }
+  }, [notifications.length]);
+
+  const navigate = useNavigate();
+  const goToFile = (groupId, fileId) =>
+    navigate(`/group/${groupId}/files/${fileId}/workstation`);
+  const goToCodeGroup = (groupId) => navigate(`/group/${groupId}`);
+
+  return (
+    <div id="Notifications">
+      <h1>Notifications</h1>
+      <div id="notifs">
+        {parsedNotifs.length > 0 &&
+          parsedNotifs.map((notif, index) => {
+            return (
+              <div
+                key={index}
+                onClick={
+                  notif.category === "added_to_code_group" ||
+                  notif.category === "file_deleted"
+                    ? () => goToCodeGroup(notif.groupId)
+                    : notif.category === "file_created" ||
+                      notif.category === "file_updated"
+                    ? () => goToFile(notif.groupId, notif.fileId)
+                    : ""
+                }
```

Comment:
I would add styling that makes it clear the notifications are clickable. From viewing the video alone I thought they weren't

Reply:
Fixed in this commit: https://github.com/izuchukwumba/peered/pull/10/commits/a885f96c46e5113c233efece82a61a2a639ca89d